docs/model-types.mdx

```diff
@@ -0,0 +1,467 @@
+A good example is the [House Sales model](/sql/tutorials/house-sales-forecasting#training-a-predictor/).
+
+</Tab>
+
+<Tab title="Case 2">
+
+Here is the [piece of code](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L279,L324) being analyzed.
+
+[If it is not a time series case, or it is a time series case with the HORIZON value being one, and the target variable type is not an array of values](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L279,L310), we use the [LightGBM](https://lightwood.io/mixer.html#mixer.LightGBM), [XGBoostMixer](https://lightwood.io/mixer.html#mixer.XGBoostMixer), [Regression](https://lightwood.io/mixer.html#mixer.Regression), and [RandomForest](https://lightwood.io/mixer.html#mixer.RandomForest) mixers.
+
+A good example is the [Home Rentals model](/sql/tutorials/home-rentals#training-a-predictor/).
+
+[Otherwise, if it is a time series case and the HORIZON value is greater than one](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L311,L324), we use the [LightGBMArray](https://lightwood.io/mixer.html#mixer.LightGBMArray) mixer.
+
+A good example is the [House Sales model](/sql/tutorials/house-sales-forecasting#training-a-predictor/).
+
+</Tab>
+
+<Tab title="Case 3">
+
+Here is the [piece of code](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L326,L351) being analyzed.
+
+[If it is an autoregressive case and the target type is an integer, float, or quantity](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L326,L351), we use the [SkTime](https://lightwood.io/mixer.html#mixer.SkTime), [ETSMixer](https://lightwood.io/mixer.html#mixer.ETSMixer), and [ARIMAMixer](https://lightwood.io/mixer.html#mixer.ARIMAMixer) mixers.
+
+The autoregressive case means that we use the target values (as encoded features) from as many previous rows as defined in the `WINDOW` clause.
+
+A good example is the [Home Rentals model](/sql/tutorials/home-rentals#training-a-predictor/).
+
+</Tab>
+
+</Tabs>
+
+<br></br>
+
+<Note>
+MindsDB may use one or multiple mixers while preparing a model. Depending on the model type and the data type of the target variable, one mixer or a set of mixers is chosen to create, train, and validate an ML model.
+</Note>
+
+The three cases above describe how MindsDB chooses the mixer candidates and stores them in the `submodels` array.
+
+By default, after collecting all relevant mixers in the `submodels` array, MindsDB uses the [BestOf](https://lightwood.io/ensemble.html#ensemble.BestOf) ensemble to [choose one of these mixers to fit the model](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L353,L358).
```

Comment:
ensemble to [single out the best mixer as the final model]

Reply:
Changed:

> By default, after training all relevant mixers in the `submodels` array, MindsDB uses the [BestOf](https://lightwood.io/ensemble.html#ensemble.BestOf) ensemble to [single out the best mixer as the final model](https://github.com/mindsdb/lightwood/blob/9a48d5523d78111c77024e1983d281f37a9782ea/lightwood/api/json_ai.py#L353,L358).